client/lib/plans/index.js

```diff
@@ -469,3 +469,14 @@ export function getPlanTermLabel( planName, translate ) {
 			return translate( 'Two year subscription' );
 	}
 }
+
+export const getPopularPlanType = siteType => {
```

Comment:
@fditrapani -- this changes the deterministic factor for determining the "popular" plan type from the `customerType` to the `siteType`.

Is that in line with what you had in mind?

Should we fall back to the current `customerType`-based behavior when the `siteType` is neither `blog` nor `professional`?  That is currently done here:

https://github.com/Automattic/wp-calypso/blob/1aaf7577e3c03d80d4c2d913a7f0f309a9a6d679/client/my-sites/plans-features-main/index.jsx#L116

...or should we promote "Business" as this currently does

Reply:
> Is that in line with what you had in mind?

Yes, that sounds right to me. How do we even determine the customer type? 

> Should we fall back to the current customerType-based behavior when the siteType is neither blog nor professional? That is currently done here:

That sounds like a good way to go.

Reply:
Customer type is not all that straightforward :)

It looks like we [sometimes render a toggle](https://github.com/Automattic/wp-calypso/blob/1aaf7577e3c03d80d4c2d913a7f0f309a9a6d679/client/my-sites/plans-features-main/index.jsx#L342) that [sometimes](https://github.com/Automattic/wp-calypso/blob/1aaf7577e3c03d80d4c2d913a7f0f309a9a6d679/client/my-sites/plans-features-main/index.jsx#L325) lets the [customer choose](https://github.com/Automattic/wp-calypso/blob/1aaf7577e3c03d80d4c2d913a7f0f309a9a6d679/client/my-sites/plans-features-main/index.jsx#L272-L293) their type.

Failing that, we [attempt to "guess" it](https://github.com/Automattic/wp-calypso/blob/1aaf7577e3c03d80d4c2d913a7f0f309a9a6d679/client/my-sites/plans-features-main/index.jsx#L399-L428).

Reply:
whao! that does look overly complicated for what? I'm not exactly sure. The code you pointed to makes me think it has to do with the tabs which makes the term customer type very suspect to me but I am a designer and I could be wrong though :D

Reply:
> makes me think it has to do with the tabs

Yes, this looks like it's related to the tabbed plans layout, in which case `customerType` is a pretty bad name - more like `planType`. Since the onboarding flow is 100%, let's use `siteType` with a premium plan fallback.

Reply:
As Cain says - that's related to tabbed plans layout and the name could be a little better. I talked to @dzver and he pointed out an important thing to keep in mind here - while during the signup we have the tabless design with plans slider, on `/plans` we still have the tabbed layout which needs to keep working for the time being. It's going to be iterated on but for now it is what it is.

Reply:
@zieladam @michaeldcain -- thanks for the context w/ the tabbed layout. Can you tell me how to test the tabbed layout?  It's not apparent to me :)

Also, so I'm clear, the new logic should be wholly based on siteType & look like this:

• `blog => TYPE_PERSONAL`
• `$else => TYPE_PREMIUM`

Should `TYPE_BUSINESS` be highlighted as "popular" for any site types?

Reply:
Adam clarified how to test the tabbed layout -- `/plans/<siteurl>` while logged in